examples/hyperparam_opt/gaussian_hyperparam_opt.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+np.random.seed(123)
+import tensorflow as tf
+tf.random.set_seed(123)
+import deepchem as dc
+import sklearn
+
+# Load delaney dataset
+delaney_tasks, delaney_datasets, transformers = dc.molnet.load_delaney()
+train, valid, test = delaney_datasets
+
+# Fit models
+metric = dc.metrics.Metric(dc.metrics.pearson_r2_score)
+
+
+def rf_model_builder(**model_params):
+  rf_params = {k: v for (k, v) in model_params.items() if k != 'model_dir'}
+  model_dir = model_params['model_dir']
+  sklearn_model = sklearn.ensemble.RandomForestRegressor(**rf_params)
+  return dc.models.SklearnModel(sklearn_model, model_dir)
```

Comment:
Here again this isn't a good example.  It will lead to people thinking their code has to be more complicated than it really does.  Let's use a standard DeepChem model for the examples.

Reply:
Swapped out for a simpler example